src/modules/fancyzones/lib/HashHelpers.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "FancyZonesDataTypes.h"
+
+namespace std
+{
+    template<>
+    struct hash<GUID>
+    {
+        size_t operator()(const GUID& Value) const
+        {
+            RPC_STATUS status = RPC_S_OK;
+            return ::UuidHash(&const_cast<GUID&>(Value), &status);
+        }
+    };
+
+    template<>
+    struct hash<FancyZonesDataTypes::DeviceIdData>
+    {
+        size_t operator()(const FancyZonesDataTypes::DeviceIdData& Value) const
+        {
+            // TODO(stefan): Implement this
```

Comment:
Well, do it then :)
It can really be anything, e.g. hash of the device name XOR GUID hash XOR hash of the monitor ID.

Reply:
Did it :)